### PR TITLE
Fix maven build

### PR DESCRIPTION
**Pull request by poperigby**

### CODE REVIEW

**Comment by learner-long-life** [comment]
Thanks @poperigby this will help us have a Java main class to run the JAR files when we're ready.
Let's avoid force-pushes in the future. I believe that arises if you don't pull and merge/rebase first before pushing.

**Comment by poperigby** [comment]
> Thanks @poperigby this will help us have a Java main class to run the JAR files when we're ready.

No problem.

>  Let's avoid force-pushes in the future. I believe that arises if you don't pull and merge/rebase first before pushing.

Yeah, that's my bad. I screwed up really bad with my commit history, but the squash should have fixed it.

File: dsa-23au/java-dsa/interval-scheduling/pom.xml [hunk 1]
```diff
@@ -49,8 +49,16 @@
           <version>2.22.1</version>
         </plugin>
```
Comment (whereismyprozac) [inline dsa-23au/java-dsa/interval-scheduling/pom.xml:50]: [Assignment and Co-Creation Week 04](https://github.com/TheEvergreenStateCollege/upper-division-cs/blob/main/dsa-23au/notes/docs/week-04/Assignment-04.md#assignment-and-co-creation-week-04)
Do all dogs go to heaven?

File: dsa-23au/java-dsa/interval-scheduling/pom.xml [hunk 2]
```diff
@@ -49,8 +49,16 @@
...
         <plugin>
-          <artifactId>maven-jar-plugin</artifactId>
-          <version>3.0.2</version>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-jar-plugin</artifactId>
```
Comment (whereismyprozac) [inline dsa-23au/java-dsa/interval-scheduling/pom.xml:53]: [Assignment and Co-Creation Week 04](https://github.com/TheEvergreenStateCollege/upper-division-cs/blob/main/dsa-23au/notes/docs/week-04/Assignment-04.md#assignment-and-co-creation-week-04)

This assignment

File: dsa-23au/java-dsa/interval-scheduling/pom.xml [hunk 3]
```diff
@@ -49,8 +49,16 @@
...
+            <version>3.2.2</version>
+            <configuration>
+                <archive>
+                    <manifest>
+                        <mainClass>dev.codewithfriends.App</mainClass>
+                    </manifest>
```
Comment (whereismyprozac) [inline dsa-23au/java-dsa/interval-scheduling/pom.xml:59]: [Assignment and Co-Creation Week 04](https://github.com/TheEvergreenStateCollege/upper-division-cs/blob/main/dsa-23au/notes/docs/week-04/Assignment-04.md#assignment-and-co-creation-week-04)

Generic comment

**Pull request merged**